python/paddle/fluid/layers/nn.py

```diff
@@ -162,8 +163,11 @@ def fc(input,
             inputs={"X": input_var,
                     "Y": w},
             outputs={"Out": tmp},
-            attrs={"x_num_col_dims": num_flatten_dims,
-                   "y_num_col_dims": 1})
+            attrs={
+                "x_num_col_dims": num_flatten_dims,
+                "y_num_col_dims": 1,
+                'use_mkldnn': use_mkldnn
```

Comment:
Save as above, this should be added in FC PR, right?

Reply:
@tensor-tang You are correct, but without those changes to FC Softmax MKLDNN would remain unused code. Till FC of MKLDNN is not ready (it is under development) then there is no other option for users to take advantage of softmax mkldnn operator  as it is usually used as activation of FC layer. Do you suggest to remove use_mkldnn attribute/param from FC ?

Reply:
I think there should be some other ways to use mkldnn softmax without change fc. But since we will implement MKLDNN FC soon, I think it's fine, you do not need revert back now.